src/itype.h

```diff
@@ -441,7 +441,7 @@ struct islot_gunmod : common_ranged_data {
     int sight_dispersion = -1;
 
     /**
-     *  If set (non-zero) mod functions as sight when recoil above mod @ref sight_dispersion */
+     *  If set (non-negative) mod functions as sight when recoil above mod @ref sight_dispersion */
```

Comment:
Wasn't it already set anyways before? If you didn't set it, it'd set to 0, which is the problem we were having with a default-set-0 disallowing the sight to apply its dispersion bonus, since being 0 meant it wasn't a sight. 

This is slightly above my paygrade but would:

` *  A non-negative value set at MAX_AIM_COST (json/itemfactory overridable) for when a mod functions as sight when recoil above mod @ref sight_dispersion */`

`    int aim_cost = 10;`

Explain it better?

And

`    /** Default sight dispersion for gun mods that function as sights */`